mxcubeweb/core/adapter/beamline_adapter.py

```diff
@@ -50,9 +50,10 @@ def gphl_json_wf_parameters_needed(self, schema, ui_schema):
         params["ui_schema"] = ui_schema
         self.app.server.emit("gphlWorkflowParametersDialog", params, namespace="/hwr")
 
-    def gphl_json_wf_update_ui_parameters(self, params):
-        # Contain only updated Values
-        self.app.server.emit("gphlWorkflowParametersDialog", params, namespace="/hwr")
+    def gphl_json_wf_update_ui_parameters(self, update_dict):
```

Comment:
We could perhaps merge `wf_parameters_*` and `gphl_json_wf_parameters_*` somehow. I dont think it should hold back this PR, but maybe as a followup